src/components/access-settings/AccessSettings.tsx

```diff
@@ -157,23 +154,43 @@ const AccessSettings: FunctionComponent<{study: Study}> = ({study}) => {
     [id]
   )
 
-  async function createNewAccount(email: string, access: Access, token: string, currentUserOrg: string) {
+  async function createNewAccount(
+    email: string,
+    firstName: string | undefined,
+    lastName: string | undefined,
+    access: Access,
+    token: string,
+    currentUserOrg: string
+  ) {
     try {
-      const {principalId, firstName, lastName} = await AccessService.getAliasFromSynapseByEmail(email)
+      let principalId: string | null = null
+      let synapseFirstName: string | null = null
+      let synapseLastName: string | null = null
+
+      if (AccessService.isSynapseEmail(email)) {
+        const synapseAlias = await AccessService.getAliasFromSynapseByEmail(email)
+        principalId = synapseAlias.principalId
+        synapseFirstName = synapseAlias.firstName
+        synapseLastName = synapseAlias.lastName
+      }
 
       const demoExternalId = await ParticipantService.signUpForAssessmentDemoStudy(token!)
 
-      await AccessService.createIndividualAccount(
+      const {identifier: userId} = await AccessService.createIndividualAccount(
         token!,
         email,
         principalId,
-        firstName,
-        lastName,
+        firstName || synapseFirstName,
+        lastName || synapseLastName,
         currentUserOrg,
         {demoExternalId},
         getRolesFromAccess(access)
       )
 
+      if (!AccessService.isSynapseEmail(email)) {
+        await UserService.sendRequestResetPassword(userId, token!)
+      }
```

Comment:
Only send a request reset password email to an account created with a non-Synapse email address

Reply:
I see, this is a new account, so there is no password, and the user cannot log in with Synapse OIDC. Thus it must be reset.

This kinda feels like logic that shouldn't be in the client, but I don't think it's necessarily a big deal--this is probably more 'agile' since this service already exists

Reply:
Agreed, not ideal -- we decided on this solution as a short-term stopgap to give study admins a self-service method to set up accounts for users with non-Synapse email addresses, until a more robust solution can be designed/implemented